lib/ui/components/cv_flat_button.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class CVFlatButton extends StatefulWidget {
+  CVFlatButton({
+    @required this.triggerFunction,
+    @required this.buttonText,
+    this.context,
+    @required Key key,
```

Comment:
Key should be the first param, just some conventions.

Reply:
@Nitish145 Check for requested changes in latest commits.